### ISSUE TITLE
Error when trying to load extensions.

### ISSUE THREAD

**Issue by AzaradoChan** [TheLastBen/fast-stable-diffusion#1738]
Since the last upgrade on collab, and the last errors with "Import fastapi" and everything else, i've excluded my extensions and today I was reinstalling them, just to see a "Permission Error", like that: 
![image](https://user-images.githubusercontent.com/65982758/224453250-c5698f76-13f8-44d8-8b4a-1717ea33e098.png)

Is there any way to solve that?

And Thank You so much.

**Comment by TheLastBen** [comment]
did that extension work before ? (promptGallery)

**Comment by AzaradoChan** [comment]
> did that extension work before ? (promptGallery)

Yes, and even extensions like PromptGenerator, OpenPose, in fact, I think, if not all of them, then almost all of them.

I would never say it's your mistake, on the contrary, thanks to you I manage to create the most bizarre and incredible images, thank you so much for giving us such a giant gift. Collab should keep previous versions too, just to make you life easier.


Google translator can sometimes misunderstand what I write, so I apologize if there are misspellings.

**Comment by AzaradoChan** [comment]
> did that extension work before ? (promptGallery)

Yes, and even extensions like PromptGenerator, OpenPose, in fact, I think, if not all of them, then almost all of them.

I would never say it's your mistake, on the contrary, thanks to you I manage to create the most bizarre and incredible images, thank you so much for giving us such a giant gift. Collab should keep previous versions too, just to make you life easier.


Google translator can sometimes misunderstand what I write, so I apologize if there are misspellings.

**Comment by TheLastBen** [comment]
try removing the extension and reinstalling them
